react/src/components/organisms/RichText/RichText.stories.js

```diff
@@ -1,76 +1,39 @@
-      };
-      const orderedOptionsWithKnobs = Object.assign(...Object.entries(unorderedOptions).map(([k, v]) => (
-        { [k]: v(UnorderedList.defaultProps[k]) })));
-      const headerIndent = boolean('headerIndent', RichText.defaultProps.headerIndent, 'RichText');
-      const anchorLinks = boolean('anchorLinks', RichText.defaultProps.anchorLinks, 'RichText');
-      const description = {
-        rteElements: [{
-          path: '@atoms/11-text/paragraph.twig',
-          data: {
-            paragraph: {
-              text: text('rteParagraph', 'Optional description', 'RichText')
-            }
-          }
-        },
-        {
-          path: '@atoms/08-lists/unordered-list.twig',
-          data: {
-            unorderedList: [
-              { text: 'This is a list item in an unordered list' },
-              { text: 'An unordered list is a list in which the sequence of items is not important. Sometimes, an unordered list is a bulleted list. And this is a long list item in an unordered list that can wrap onto a new line.' },
-              {
-                text: 'Lists can be nested inside of each other',
-                sublist: [
-                  { text: 'This is a nested list item' },
-                  { text: 'This is another nested list item in an unordered list' }
-                ]
-              },
-              { text: 'This is the last list item' }
-            ]
-          }
-        }]
+      const props = {
+        rawHtml: text('rawHtml', '<h1>This is heading 1</h1>'),
+        className: text('className', ''),
+        id: text('id', ''),
+        htmlTag: text('htmlTag', 'div'),
+        transform: action('transform callback')
       };
       return(
-        <RichText headerIndent={headerIndent} anchorLinks={anchorLinks} {...description}>
-          <CompHeading {...compOptionsWithKnobs} />
-          <SidebarHeading {...sideOptionsWithKnobs} />
-          <Paragraph {...paraOptionsWithKnobs} />
-          <Paragraph text={text('Paragraph2: text', 'foobarbaz', 'Paragraph')} />
-          <Heading text={text('Heading: text', 'This is a heading', 'Heading')} level={1} />
-          <UnorderedList {...orderedOptionsWithKnobs} />
-          <DecorativeLink {...decoreOptionsWithKnobs} />
-        </RichText>);
+        <RichText {...props}>
+          <h1>This is a heading.</h1>
+          <p>
+          An <b>unordered list</b> is a list in which the sequence of items is not important. Sometimes, an unordered list is a bulleted list. And this is a long list item in an unordered list that can wrap onto a new line.
```

Comment:
My IDE is saying this indent is incorrect (should be 2 more), but seems eslint passed anyways?